src/main/java/racing/domain/RacingGame.java

```diff
@@ -0,0 +1,81 @@
+package racing.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class RacingGame {
+
+    private static final int RANGE_OF_RANDOM_NUMBER = 10;
+
+    private final List<Car> cars;
+
+    public RacingGame() {
+        this.cars = new ArrayList<>();
+    }
+
+    public void startGame() {
+        try {
+            final List<String> nameList = UserInput.getCarNames();
+            for (String name : nameList) {
```

Comment:
아래와 같이 변수명을 변경하는 것이 더 좋을 것 같아요. 😉

```suggestion
            final List<String> names = UserInput.getCarNames();
            for (String name : names) {
```

Reply:
앗!!!!!!!!! 왜 저렇게 변수명을 지었는지 아직도 의문이네요... 피드백 감사합니다! 😄